docker-compose.yml

```diff
@@ -4,13 +4,12 @@ services:
 
   backend:
     build:
-      context: ./backend
-      dockerfile: Dockerfile
+      context: ./
```

Comment:
Why does the backend container need access to everything if you copy the frontend stuff into it manually?

Reply:
During the first stage build it needs access to the frontend folder